packages/language-server/src/plugins/html/features/astro-attributes.ts

```diff
@@ -42,6 +42,7 @@ export const astroAttributes = newHTMLDataProvider('astro-attributes', {
 		{
 			name: 'is:raw',
 			description: 'Instructs the Astro compiler to treat any children of this element as text',
+			valueSet: 'v',
```

Comment:
Unrelated to the config change, I just noticed that it had the wrong valueSet while testing things. This is very much harmless so figured I'd do it here